romana-install/group_vars/aws

```diff
@@ -7,5 +7,18 @@ instance_type:
     devstack: "t2.small"
     kubernetes: "t2.small"
 
-controller_image: "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
-compute_image: "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
+distro_info:
+  "ubuntu":
+    ami:  "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
+    user: "ubuntu"
+    bootstrap:
+      - apt-get install python-apt
+      - printf '%s\n' '{{ lookup("file", romana_ssh_key + ".pub") }}' >> /home/ubuntu/.ssh/authorized_keys
+  "centos":
```

Comment:
I can see already the additional distros in the future: RHEL, Fedora, BSD, Solaris, WinNT... :-)